sqldelight-studio-plugin/build.gradle

```diff
@@ -10,9 +10,10 @@ targetCompatibility = JavaVersion.VERSION_1_6
 
 intellij {
   updateSinceUntilBuild false
-  version 'IC-143.870.1'
```

Comment:
yea. i'll change it to a current official build that works. I was running in to issues with tests running because the kotlin runtime was missing in the earlier versions of idea. This is an easy remedy (there might be a more complicated one).